lab_3_generate_by_ngrams/target_score.txt

```diff
@@ -1 +1 @@
-10
+10
```

Comment:
?

Reply:
I occasionally changed target score in the wrong lab